app/client/rest/posts.ts

```diff
@@ -11,11 +11,11 @@ export interface ClientPostsMix {
     getPost: (postId: string) => Promise<Post>;
     patchPost: (postPatch: Partial<Post> & {id: string}) => Promise<Post>;
     deletePost: (postId: string) => Promise<any>;
-    getPostThread: (postId: string) => Promise<any>;
-    getPosts: (channelId: string, page?: number, perPage?: number) => Promise<PostResponse>;
-    getPostsSince: (channelId: string, since: number) => Promise<PostResponse>;
-    getPostsBefore: (channelId: string, postId: string, page?: number, perPage?: number) => Promise<PostResponse>;
-    getPostsAfter: (channelId: string, postId: string, page?: number, perPage?: number) => Promise<PostResponse>;
+    getPostThread: (postId: string, collapsedThreads?: boolean, collapsedThreadsExtended?: boolean) => Promise<any>;
```

Comment:
Most of the places where I have seen using it, we use for both the same variable (`isCRTEnabled`). Do we need them to be two different variables?

Reply:
Yes, on mobile we are always getting the extended data to avoid fetching related data again (Eg: user data). I have kept the parameters to keep it separate and configurable if using it for the other use cases.